clientgui/BOINCTaskBar.cpp

```diff
@@ -433,7 +435,10 @@ bool CTaskBarIcon::SetIcon(const wxIcon& icon, const wxString& ) {
     else if (m_iconTaskBarSnooze.IsSameAs(icon))
         macIcon = wxImage(macsnoozebadge);
     else {
-        err = SetDockBadge(NULL);
+#if wxDEBUG_LEVEL
+        err = 
```

Comment:
The same as described above: I suggest to add proper error handling in both Debug and Release modes

Reply:
Again, this is not my code. Rom Walton wrote it many years ago. While I agree with you, it is well beyond the scope of this PR and not something I am willing to do. If you feel strongly, I suggest you create a separate issue.